tests/reftests/pin.test

```diff
@@ -175,25 +996,101 @@ echo "another-inexistant" "inexistant"
 -> installed qux.dev
 -> installed bar.dev
 Done.
-### : Test opam pin remove <pkg>.<version>
-### opam pin remove --no-action bar.dev
-Ok, bar is no longer pinned to file://${BASEDIR}/bar (version dev)
+### :XI:c: Test opam pin remove <pkg>.<version>
+### opam switch create removal --empty
+### opam pin ./nip --no-action
+nip is now pinned to file://${BASEDIR}/nip (version ved)
+### opam pin remove --no-action nip.dev
+[ERROR] nip is pinned but not to version dev. Skipping.
+# Return code 2 #
+### opam pin ./nip --no-action
+[NOTE] Package nip is already pinned to file://${BASEDIR}/nip (version ved).
+nip is now pinned to file://${BASEDIR}/nip (version ved)
 ### opam pin remove --no-action nip.wrong-version
 [ERROR] nip is pinned but not to version wrong-version. Skipping.
 # Return code 2 #
-### : Test opam pin remove --all
-### OPAMNODEPEXTS=1
-### opam pin add --no-action bar.dev ./bar
+### opam pin remove --no-action nip2.dev
+[NOTE] nip2 is not pinned.
+### :XI:d: Test opam pin remove --all
```

Comment:
I kept this test here, "as is": be the most similar to previous one with several kind of pins, etc. I don't know if it is intended or not?

Reply:
i'm not sure to understand what you mean here

Reply:
Is it intended to have different kind of pins to remove ? or it was the case because the current switch had already several kind of pins?